code/game/objects/items/devices/radio/radio.dm

```diff
@@ -746,3 +754,82 @@ var/global/list/default_medbay_channels = list(
 /obj/item/device/radio/phone/medbay/New()
 	..()
 	internal_channels = default_medbay_channels.Copy()
+
+/obj/item/device/radio/random_radio
+	name = "Random wave radio"
+	desc = "Radio that can pick up message from secure channels, but with small chance. It can be repaired by atifact, that have knowledge of mechanics."
+	icon = 'icons/obj/faction_item.dmi'
+	icon_state = "random_radio"
+	item_state = "random_radio"
+	slot_flags = FALSE
+	canhear_range = 4
+	var/random_hear = 20
+	channels = list("Command" = 1, "Security" = 1, "Engineering" = 1, "NT Voice" = 1, "Science" = 1, "Medical" = 1, "Supply" = 1, "Service" = 1, "AI Private" = 1)
+	price_tag = 20000
```

Comment:
Price is way too high for the utility it provides, could be solved by IC means as guild not wanting to export their own items, or by cuting the price.

Reply:
More utility added.